tcms/static/js/testrun_actions.js

```diff
@@ -22,6 +23,8 @@ function toggleDiv(link, divId) {
 Nitrate.TestRuns.Details.on_load = function() {
   setAddTagAutocomplete();
 
+  jsonRPC('TestExecutionStatus.filter', {}, data => executionStatuses = data)
```

Comment:
There is a `jQ(document).ready(function(){` in this JS file. I believe this line is better suited in there.